packages/Config/RectorConfig.php

```diff
@@ -223,7 +223,20 @@ static function (
 
     public function import(string $filePath): void
     {
-        $this->importFile($filePath);
+        if (str_contains($filePath, '*')) {
+            throw new ShouldNotHappenException(
+                'Matching file paths by using glob($filePath) is no longer supported. Use specific file path instead.'
+            );
+        }
+
+        Assert::fileExists($filePath);
```

Comment:
not sure we need this fileExists call, as the require below will fail hard if the path is missing

Reply:
thats for throw exception for file not exists message consistency